projects/boringssl/build.sh

```diff
@@ -1,6 +1,5 @@
-#!/bin/bash -eux
-#
-# Copyright 2016 Google Inc.
+#!/bin/bash -eu
+# Copyright 2020 Google Inc.
```

Comment:
I think you can (read: should) revert these.

It's unclear why removing the `-x`, but also, we try to avoid copyright year bumps :)